e2e/scenario/vhd.go

```diff
@@ -164,6 +165,11 @@ func (c *VHDCatalog) Ubuntu2204Gen2Containerd() VHD {
 	return c.Ubuntu2204.Gen2Containerd
 }
 
+// Returns the gen2containerdprivatekubepkg catalog entry.
+func (c *VHDCatalog) Ubuntu2204Gen2ContainerdPrivateKubePkg() VHD {
+	return c.Ubuntu2204.Gen2Arm64ContainerdPrivateKubePkg
```

Comment:
is this actually an ARM64-based VHD?

Reply:
Either will work for this test but correcting the naming.